packages/nimble-components/src/menu/styles.ts

```diff
@@ -48,4 +57,17 @@ export const styles = css`
         padding-top: ${smallPadding};
         padding-bottom: ${smallPadding};
     }
-`;
+`.withBehaviors(
+    themeBehavior(
+        Theme.color,
+        css`
+            :host::before {
```

Comment:
I dunno if the pseudoelements video we watched this week had anything to do with you coming up with this solution, but it at least helped me understand it. Cool approach! Maybe we can use pseudoelements more often when we find ourselves needing one extra element just to add borders/backgrounds to match a design spec.

Reply:
@rajsite actually mentioned that what could probably use `::before` and `::after` to solve some of these styling scenarios the day before we watched the video, so the timing of everything aligned to make me want to try it out.

I noticed that the combo-box & select have the same problem, so assuming we're happy with this solution, I'll post a follow-up PR to fix those components. I'm hoping we can also leverage this approach for some table styling.

Reply:
If we agree on it as a pattern then it could be helpful to describe it in our [CSS Guidelines](https://github.com/ni/nimble/blob/main/packages/nimble-components/docs/css-guidelines.md). That can obviously wait for the follow up PR.